packages/story-editor/src/components/canvas/karma/quickActions.karma.js

```diff
@@ -23,7 +23,7 @@ import { TEXT_ELEMENT_DEFAULT_FONT } from '@googleforcreators/elements';
  * Internal dependencies
  */
 import { useStory } from '../../../app';
-import { ACTIONS } from '../../../app/highlights';
+import { ACTIONS } from '../../../app/quickActions';
```

Comment:
Here, too -- curious why the change -- moving some files into `quickActions` folder instead.

Reply:
This was an ask from @swissspidy

Reply:
Ask is maybe the wrong word, but it definitely didn't feel right that `quickActions` is nested within `highlights`. It could easily be one level up, so I suggested that once. Makes it easier to separately convert `quickActions` later (see #12705)

Reply:
True, it was quite confusing before.